CONTRIBUTING.md

```diff
@@ -36,3 +32,27 @@ Additionally, for the sake of simplicity, we are standardizing on what should be
 
 ![screen shot 2016-08-16 at 8 03 22 pm](https://cloud.githubusercontent.com/assets/777633/17719999/920fb534-63ec-11e6-8903-3725f2cd05b0.png)
 ![screen shot 2016-08-16 at 8 03 36 pm](https://cloud.githubusercontent.com/assets/777633/17719998/920cff6a-63ec-11e6-8f76-0ac7a5fa0c9d.png)
+
+## Best Practices for Contributing
+[Best Practices for Contributing]: #best-practices-for-contributing
+* Before you start coding, open an issue so that the community can discuss your change to ensure it is in line with the goals of the project and not being worked on by someone else. This allows for discussion and fine tuning of your feature and results in a more succent and focused additions.
```

Comment:
I think we went over this being a tad too much for someone that is only making a one line change in #156. Because they could just explain it in the PR. However if the change is large enough, then an issue should be created in an RFC kind of style? I'm just not certain of the best way to word that.